ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -149,6 +176,7 @@ def change_bgp_dut_state(self, is_up=True):
                 self.fails['dut'].add('Stderr: %s' % stderr)
 
     def verify_bgp_dut_state(self, state='Idle'):
+        state = state.split(',')
```

Comment:
>state [](start = 8, length = 5)

better not to reuse var, and name it as 'states'

Reply:
done